src/test/java/org/svydovets/baseEntity/Note1.java

```diff
@@ -0,0 +1,22 @@
+package org.svydovets.baseEntity;
+
+import org.svydovets.annotation.Entity;
+import org.svydovets.annotation.Id;
+import org.svydovets.annotation.JoinColumn;
+import org.svydovets.annotation.ManyToOne;
+import org.svydovets.annotation.Table;
+
+@Entity
+@Table(name = "notes")
+public class Note1 {
```

Comment:
We need more clear names for test entities, I have no idea what does "Note1" mean. This entity doesn't have a "@JoinColumn" annotation, so we can name it like "NoteWithoutJoinColumnAnnotation" or "NoteWithoutJoinColumnAnnotationForPerson"